app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -334,13 +346,12 @@ class NavigationOverlayFragment : Fragment() {
         pinnedTileViewModel.getTileList().observe(viewLifecycleOwner, Observer {
             if (it != null) {
                 tileAdapter.setTiles(it)
-                updateFocusableViews()
             }
         })
 
         adapter = tileAdapter
 
-        layoutManager = HomeTileManager(context, COL_COUNT)
+        layoutManager = GridLayoutManager(context, COL_COUNT)
```

Comment:
This seems unrelated to the other changes, what is it for?

Reply:
HomeTileManager has custom focusUpId logic that depends on PocketState.